examples/README.md

```diff
@@ -0,0 +1,37 @@
+# Examples
+
+This directory contains examples for how to use the Planet Python Client.
+
+# Navigation
+Each example script is named according to the convention
+`<Planet API>_<use_case>.py`.
+
+## Usage
+
+To use these examples, just run the scripts with `python <script_name>`. These
+examples hit the Planet servers, so be aware that usage and fees may apply.
+
+## Contributing
```

Comment:
Oooh, this might be nice to add to a CONTRIBUTING.md!

Reply:
Great suggestion. Added a reference to this doc in CONTRIBUTING.md That prompted a whole update of CONTRIBUTING.md, which brings it up to date with nox usage and testing docs/examples. Take a look if you get a chance.